source/_posts/2024-01-25-matter-livestream-blog.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "The State of Matter"
+description: "Catch up on all of the highlights of the State of Matter livestream that we streamed on Wednesday, January 10, 2024."
+date: 2024-01-25 00:00:01
+date_formatted: "January 25, 2024"
+author: Marcel van der Veldt
+comments: true
+categories: Matter
+og_image: /images/blog/2024-01-matter/matter.png
+---
+During our State of Matter live stream two weeks ago, we explained what Matter is, how it relates to Thread, and how you can compare it with existing protocols you may already know, like Zigbee or Z-Wave. 
+
+In this blog post, we’ve extracted the highlights of the stream for you. We also link to our updated [Matter](/integrations/matter/) and [Thread](/integrations/thread/) documentation where relevant.
+
+You can watch the entire live stream here:
+
+<lite-youtube videoid="rEugjMk-4II" videotitle="State of Matter"></lite-youtube>
+
+We’ll also highlight each segment of the live stream in this blog under each corresponding heading, so you can start watching the specific parts that interest you!
+
+<!--more-->
+## Why we believe in Matter
+<lite-youtube videoid="rEugjMk-4II" videoStartAt="213" videotitle="State of Matter"></lite-youtube>
+We believe in Matter: it’s open source, and most importantly, it’s fully local by default. Matter will allow us to control devices ranging from lights, to robot vacuums, to TVs, and to many other IP-connected devices, all through a standardized protocol. It is a huge step forward towards having more sustainable and worry-free smart home products. For this reason, Nabu Casa uses the revenue we get from the subscribers to Home Assistant Cloud (thank you all!) to employ developers who are dedicated to implementing Matter. And we have even joined the Connectivity Standards Alliance (CSA) - the organization behind the Matter standard - as a participant to have a front-row seat and to defend the [Open Home values](https://www.home-assistant.io/blog/2021/12/23/the-open-home/) during the development of the standard.
```

Comment:
This is risky, as we are switching HTML & Markdown context here. 

We need to add empty lines between them multiple times to allow the parser to switch. In this case, it seems to be handled, but it isn't guaranteed.

```suggestion
<!--more-->

## Why we believe in Matter

<lite-youtube videoid="rEugjMk-4II" videoStartAt="213" videotitle="State of Matter"></lite-youtube>

We believe in Matter: it’s open source, and most importantly, it’s fully local by default. Matter will allow us to control devices ranging from lights, to robot vacuums, to TVs, and to many other IP-connected devices, all through a standardized protocol. It is a huge step forward towards having more sustainable and worry-free smart home products. For this reason, Nabu Casa uses the revenue we get from the subscribers to Home Assistant Cloud (thank you all!) to employ developers who are dedicated to implementing Matter. And we have even joined the Connectivity Standards Alliance (CSA) - the organization behind the Matter standard - as a participant to have a front-row seat and to defend the [Open Home values](https://www.home-assistant.io/blog/2021/12/23/the-open-home/) during the development of the standard.
```